src/ModelFinder.php

```diff
@@ -46,14 +46,17 @@ protected static function fullQualifiedClassNameFromFile(
         string $basePath,
         string $baseNamespace
     ): string {
+
+        $_app = ($baseNamespace != '') ? '' : app()->getNamespace();
```

Comment:
Let's just update the value of `$baseNamespace`

Could you make use of the null assignment operator to clean up the code?